packages/KSkeleton/__snapshots__/KSkeleton.spec.js.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`KSkeleton matches snapshot 1`] = `undefined`;
```

Comment:
looks like the snapshot test isn't useful here since maybe the skeleton isn't rendered because of delayMilliseconds?